migrate/cmd/migrate/main.go

```diff
@@ -212,23 +212,18 @@ func runCmd(req commandRequest) (string, error) {
 		sourceURL = fmt.Sprintf("file://%s", sourceURL)
 	}
 
-	schema := req.Schema
```

Comment:
I think this is still relevant

Reply:
This variable exists so that the following conditional can force schema to be `app_config` when gateway migration is running. It seems strange to do it, and I don't think it is needed?